redash/query_runner/python.py

```diff
@@ -187,9 +184,9 @@ def run_query(self, query):
             #       One option is to use ETA with Celery + timeouts on workers
             #       And replacement of worker process every X requests handled.
 
-            exec(code) in restricted_globals, script_locals
+            exec(code) in restricted_globals, self._script_locals
 
-            json_data = json.dumps(self._result)
```

Comment:
Actually it's better to accumulate the log statements in another variable, and then put them in result["log"], or they might overriden -- I'll do it in this branch.